### ISSUE TITLE
HP firmware update script does no longer patch 561T models correctly

### ISSUE THREAD

**Issue by BjoernT** [rcbops/openstack-ops#151]
The Intel 10g NIC (HP 561T Models) are no longer supported since intel nic firmware 1.17 and the older cards are not always reporting as unpatched.
The firmware script needs to be updated to use the 1.16.1 firmware for older NIC models and 1.19.1 for all other models

**Comment by rax-rstark** [comment]
Ran into this issue this past weekend. Thanks for having it documented.

**Comment by BjoernT** [comment]
Yes thats why I downgraded the firmware at the latest version of the script so I can add this multi firmware fix

**Comment by rax-rstark** [comment]
New version released from HPE on 2021/10/29: 

https://support.hpe.com/hpesc/public/swd/detail?swItemId=MTX_a370c322c0ff46a4918a7a8537

firmware-nic-intel-1.22.6-1.1.x86_64.rpm

**Comment by spfng** [comment]
Dear friend, can you share please hp-firmware-system-p89-3.02_2022_07_18-1.1.i386.rpm this file? Telegram @spfng
